### PR TITLE
AI Sprites, Vox clothing sprites, click AI display as AI to interact with it

### PR DESCRIPTION
**Pull request by uraniummeltdown**
https://github.com/tgstation/tgstation/pull/22515
Added AI sprites from TG, dead AIs now get a death sprite that matches the living one.
Added Vox clothing sprites from /vg/
Click AI display as AI to change it.

out

voxvgsprites

:cl:
add: Added new AI core sprites from TG
add: Added Vox rad suit, bombsuit hood, paramedic EVA suit sprites from /vg/
tweak: You can click AI displays as an AI to change them
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
Pictures of the sprites?

**Comment by Alexshreds** [comment]
Number 1 rule of sprite PRs is pics or it didn't happen

**Comment by Crazylemon64** [comment]
This PR kind of feels all over the place
"Adds interaction to ore box and adv mining scanner" seems rather disparate from "Adds new AI core sprites". Could all these things be split up, perhaps?

**Comment by uraniummeltdown** [comment]
Sorry I got a bit carried away, although I don't believe any of the changes are controversial (if they are I will be happy to split them off). In future I'll avoid the mega PR format

**Comment by KasparoVy** [comment]
It would be much appreciated to see the Vox/AI sprites as they are in game (with the different dirs as applicable).

**Comment by Fox-McCloud** [comment]
Yeah, there's a bit much in this PR, as Crazy noted; it's kinda...all over; mostly the sprites.

**Comment by fludd12** [comment]
I like all the changes, but having them all and changing them all in separate commits at the very least would be preferable, I think - if not full PRs for each of them.

**Comment by taukausanake** [comment]
If you can construct xenobip boards and machines now (and I'm thinking the computer ones that teleport everything) then I request they are removed from xenobio at round start so that they are actually researched and built for usage. Otherwise you have really no reason to ever build them unless you're making replacements but if then you might not even return to a nuked xenobio

**Comment by KasparoVy** [comment]
Thanks for the pics

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -33,11 +33,9 @@
 	species_restricted = list("exclude", "Diona", "Wryn")
 	species_fit = list("Vox")
 	sprite_sheets = list(
-		"Vox" = 'icons/mob/species/vox/suit.dmi',
-		"Drask" = 'icons/mob/species/drask/suit.dmi'
```
Comment (Fox-McCloud) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: you removed drask?

Comment (uraniummeltdown) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: There actually is no Drask icon for the cap's space suit

Comment (Fox-McCloud) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Thanks!

**Comment by DesolateG** [comment]
Is the third from the bottom on the right supposed to be Cabal from Command and Conquer?

**Pull request merged**